src/main/scala/org/globalforestwatch/summarystats/forest_change_diagnostic/ForestChangeDiagnosticSummary.scala

```diff
@@ -83,6 +84,37 @@ object ForestChangeDiagnosticSummary {
             0
           }
         }
+
+        val gfwProCoverage: Map[String, Boolean] =
+          raster.tile.gfwProCoverage.getData(col, row)
+        val argPresence = gfwProCoverage.getOrElse("Argentina", false)
+        val braBiomesPresence = gfwProCoverage.getOrElse("Brazil Biomes", false)
+
+        // We compute country-specific forest loss using argForestLoss tile for
+        // Argentina, and prodesLossYear for Brazil. In the very unusual case where a
+        // location covers more than one country, we don't want to mix
+        // country-specific forest losses, so we record the country-code that the
+        // forest loss came from. We will zero out the country-specific forest loss
+        // and mark it with an 'ERR' country code if we end up merging results from
+        // more than one country.
+        val (countrySpecificLossYear: ApproxYear, countryCode: String) =
```

Comment:
might be more verbose but a little easier to read if you just assign these each on individual lines in the if statements instead of returning a val as a tuple, but I could go either way

Reply:
I was trying to go with the Scala functional style, but I agree the imperative style is easier to read.  So, did your suggested changes!